dump-scripts/python/zend_to_horizen.py

```diff
@@ -25,54 +44,93 @@
 def satoshi_2_wei(value_in_satoshi):
 	return int(round(SATOSHI_TO_WEI_MULTIPLIER * value_in_satoshi))
 
-if len(sys.argv) != 3:
+if len(sys.argv) != 3 and len(sys.argv) != 5:
 	print(
-		"Usage: python3 {} <zend dump file name> <output_file>"
+		"Usage: python3 {} <zend dump file name> <mapping file name> <zend_vault_output_file> <eon_vault_output_file>"
```

Comment:
i would rename 
<eon_vault_output_file>
to
<eon_vault_automappings_file>

Reply:
Done